xbmc/video/VideoDatabase.cpp

```diff
@@ -4905,11 +4911,22 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE settings ADD VideoStream integer");
     m_pDS->exec("ALTER TABLE streamdetails ADD strVideoLanguage text");
   }
+
+  if (iVersion < 104)
+  {
+    m_pDS->query(PrepareSQL("SELECT GROUP_CONCAT(episode.idEpisode), episode.c%02d, COUNT(episode.idFile) FROM episode JOIN streamdetails ON streamdetails.idFile = episode.idFile AND streamdetails.iStreamType = 0 WHERE episode.c%02d = streamdetails.iVideoDuration GROUP BY episode.idFile HAVING COUNT(episode.idFile)>1", VIDEODB_ID_EPISODE_RUNTIME, VIDEODB_ID_EPISODE_RUNTIME));
```

Comment:
Could you break this up so it's a little bit easier to read? But not really a blocker.